spec/features/challenge_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Challenges', type: :feature do
```

Comment:
Nice job writing a test for this homepage too - not needed but is good practice!